components/storage/StorageTable.tsx

```diff
@@ -79,25 +79,52 @@ const Table: React.FC<TableProps> = ({ services, selectedAnswers, questions }) =
   const columnHelper = createColumnHelper<TableRow>();
   const tableContainerRef = useRef<HTMLDivElement>(null);
 
-const tableData: TableRow[] = useMemo(() => {
-  const uniqueFeatureNames = new Set<string>();
-  services.forEach(service => {
+  // --- FEATURE SORTING LOGIC ---
+  const sortedFeatureNames = useMemo(() => {
+    const featureNamesFromForm = questions.map(q => q.affected_feature);
+    const uniqueServiceFeatureNames = new Set<string>();
+    services.forEach(service => {
       service.features?.forEach(feature => {
-          uniqueFeatureNames.add(feature.name);
+        uniqueServiceFeatureNames.add(feature.name);
       });
-  });
+    });
```

Comment:
Is the list of features (i'm assuming you mean the rows) not the same for all of the services? We could just define that all of the services need to list something for the feature. And then you wouldn't need to create a set

Reply:
it is the same for the services that have features, but Globus does not have any features. i wanted to leave Globus in `storage-tool.yaml` as a service, rather than putting it in a separate markdown file. Globus should not be in the table, because it is not a service per say. The way I remove it from the table is removing it from the table if it doesnt have any features. maybe it makes more sense to have it in it's own markdown file?